### PR TITLE
Add Fr-scan.com source to Madara

### LINKED ISSUE #14741: fr-scan.com source request
### Source name

fr-scan

### Source link

https://fr-scan.com

### Language

French

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a title with source name.
- [X] I have checked that the extension does not already exist on the [website extensions list](https://tachiyomi.org/extensions/) or the app.
- [X] I have checked that the extension does not already exist by searching the [GitHub repository](https://github.com/tachiyomiorg/tachiyomi-extensions/) and verified it does not appear in the code base.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by KirinRaikage**
Add Fr-scan source

Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

Closes #14741

### CODE REVIEW

File: .github/workflows/issue_moderator.yml [hunk 1]
```diff
@@ -58,6 +58,12 @@ jobs:
                 "regex": ".*shinigami\\.id.*",
                 "ignoreCase": true,
                 "message": "Duplicate of #15408"
+              },
+              {
+                "type": "body",
+                "regex": ".*olympusscans\\.com.*",
```
Comment (alessandrojean) [inline .github/workflows/issue_moderator.yml:64]: You probably need to rebase your branch on top of master. This shouldn't be included in this PR.

File: multisrc/overrides/madara/frscan/src/FRScan.kt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package eu.kanade.tachiyomi.extension.fr.frscan
+
+import eu.kanade.tachiyomi.multisrc.madara.Madara
+import java.text.SimpleDateFormat
+import java.util.*
```
Comment (alessandrojean) [inline multisrc/overrides/madara/frscan/src/FRScan.kt:5]: Don't use widecard imports, it will make the lint to fail later.

**Review by alessandrojean — CHANGES_REQUESTED** [review]

**Comment by KirinRaikage** [comment]
Done for both changes requested, thanks for the review!

File: multisrc/overrides/madara/frscan/src/FRScan.kt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+package eu.kanade.tachiyomi.extension.fr.frscan
+
+import eu.kanade.tachiyomi.multisrc.madara.Madara
+import java.text.SimpleDateFormat
```
Comment (alessandrojean) [inline multisrc/overrides/madara/frscan/src/FRScan.kt:4]:
```suggestion
import java.text.SimpleDateFormat
import java.util.Locale
```

**Review by alessandrojean — CHANGES_REQUESTED** [review]

**Review by alessandrojean — APPROVED** [review]
